libpod/buildah/buildah.go

```diff
@@ -0,0 +1,232 @@
+package buildah
+
+import (
+	"encoding/json"
+	"path/filepath"
+
+	is "github.com/containers/image/storage"
+	"github.com/containers/image/types"
+	"github.com/containers/storage"
+	"github.com/containers/storage/pkg/ioutils"
+	"github.com/opencontainers/go-digest"
+	"github.com/opencontainers/image-spec/specs-go/v1"
+	"github.com/pkg/errors"
+	"github.com/projectatomic/libpod/cmd/podman/docker"
+)
+
+const (
+	// Package is the name of this package, used in help output and to
+	// identify working containers.
+	Package = "libpod"
```

Comment:
Did we patch this? Should we have patched this? I'm wondering if this will affect the containers buildah sees

Reply:
i changed it if that is what you are asking

Reply:
reverted